### PR TITLE
added transient fields management

### PR DESCRIPTION
**Pull request by remievaneos**
Add transient field by giving the list to the writer when creating it

### CODE REVIEW

File: src/Berthe/DAL/AbstractWriter.php [hunk 1]
```diff
@@ -97,6 +99,15 @@ public function setTableName($tableName)
 
         return $this;
     }
+    
+    /**
+     * Set the transient fields (ie. fields that won't be saved in / retrieved from database)
+     * 
+     * @param array $transientFields
```
Comment (christophehurpeau) [inline src/Berthe/DAL/AbstractWriter.php]: `@param string[]`

File: src/Berthe/DAL/AbstractWriter.php [hunk 2]
```diff
@@ -269,13 +280,14 @@ public function deleteById($id) {
...
     }
-
+    
     protected function getDefaultMappings(\Berthe\VO $vo) {
         $properties = array_keys($vo->__toArray());
         $mappings = array_combine($properties, $properties);
-
-        return array_filter($mappings, function($value) {
-            return ! ($value == 'id' || $value == 'version');
+        $self = $this;
```
Comment (christophehurpeau) [inline src/Berthe/DAL/AbstractWriter.php]: why not transientFields directly ?

Comment (remievaneos) [inline src/Berthe/DAL/AbstractWriter.php]: Pas compris...

Comment (maffpool) [inline src/Berthe/DAL/AbstractWriter.php]: use transientFields directement plutôt que use self
Je pense que c'est ce que @christophehurpeau a voulu dire

**Pull request merged**